### ISSUE TITLE
If you pass to `outDir` absolute path in will convert it to relative and crash.

### ISSUE THREAD

**Issue by cawa-93** [jbaubree/sitemap-ts#52]
If you pass to `outDir` absolute path in will convert it to relative and crash.

```js
    generateSitemap({
      outDir: 'C:/Users/kozac/Dev/kozack.me/docs/.vitepress/dist',
      dynamicRoutes: pages,
    })
```
```
Error: ENOENT: no such file or directory, open 'C:\Users\kozac\Dev\kozack.me\C:\Users\kozac\Dev\kozack.me\docs\.vitepress\dist\robots.txt'
    at Object.openSync (node:fs:599:3)
    at Object.writeFileSync (node:fs:2221:35)
    at generateSitemap (C:\Users\kozac\Dev\kozack.me\node_modules\sitemap-ts\dist\index.cjs:122:6)
    at Object.buildEnd (C:\Users\kozac\Dev\kozack.me\docs\.vitepress\config.js:72:43)
    at build (file:///C:/Users/kozac/Dev/kozack.me/node_modules/vitepress/dist/node/serve-1b26f7f9.js:41479:59)
```

**Comment by jbaubree** [comment]
Hey, thanks for reporting ! I will fix it as soon as possible

**Comment by jbaubree** [comment]
@cawa-93 fixed in `v1.1.0` :)
